### ISSUE TITLE
Implement a way to create closed comments

### ISSUE THREAD

**Issue by gloompi** [WPO-Foundation/webpagetest-github-action#6]
Currently, webpagetest action creates a comment on each commit and it gets polluted really fast, can we create closed comments, and open them whenever we need to actually read a waterfall, or another approach would be deleting the old comment whenever we add a new one.

**Comment by tkadlec** [comment]
@gloompi Are you thinking something like this?

![Screen Shot 2021-09-30 at 10 15 02 AM](https://user-images.githubusercontent.com/66536/135483069-73dd7c21-4ecd-4edc-bb54-4ba2559c3edb.png)

**Comment by gloompi** [comment]
@tkadlec yes, is there such an option to make comments closed by default?

**Comment by tkadlec** [comment]
There isn't, but this seems like a good thing for us to add.
